### DISCUSSION TITLE
Is WinUI 3 for c++ still supported and will bugs still be fixed?

### DISCUSSION THREAD

**Discussion by yki5000** [microsoft/WinUI-Gallery#1333] [Q&A]
Sorry, if I have to ask a stupid question, but I'm not a professional programmer now and you read something here and there. I keep discovering bugs with standard controls in WinUi 3 in C++. Is this (officially) still supported and also fixed?

(I've already noticed that the priority is probably C#?!)

Regards

**Comment by marcelwgn** [comment]
While I'm not working for Microsoft, I think it's fair to say that WinUI 3 for C++ is still supported and bugs will be fixed. Given that this repository is for the WinUI Gallery and not WinUI itsself, if you find any bugs feel free to open an issue on the [WinUI repository](https://github.com/microsoft/microsoft-ui-xaml)
